src/shared/components/query/QueryStore.ts

```diff
@@ -969,7 +969,15 @@ export class QueryStore
 	@computed get defaultSelectedSampleListId()
 	{
 		if (this.isVirtualCohortQuery) {
-			return ALL_CASES_LIST_ID;
+			let toReturn = ALL_CASES_LIST_ID;

```

Comment:
this method really needs some tests.  this would a good opportunity to add them, at least for this logic here.

Reply:
you can put the body of htis method in a helper function and then call it here.